.github/workflows/chatbot-test.yml

```diff
@@ -0,0 +1,24 @@
+name: Chat Bot Test
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/**'
```

Comment:
point to `.github/workflows/chatbot-test.yml` `.github/workflows/chatbot-inference-llama-2-7b-chat-hf.yml`

Reply:
done